spec/system/company_values_crud_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'company values CRUD actions process', type: :system do
+  let(:admin) { create(:admin) }
+  let!(:companyvalue) { create(:company_value) }
+
+  before do
+    sign_in admin
+    visit admins_company_values_path
+  end
+
+  it 'create action' do
```

Comment:
:) and same for the others
```suggestion
  it 'creates action' do
```